x-pack/plugins/security_solution/public/alerts/pages/detection_engine/rules/types.ts

```diff
@@ -10,6 +10,15 @@ import { Filter } from '../../../../../../../../src/plugins/data/common';
 import { FormData, FormHook } from '../../../../shared_imports';
 import { FieldValueQueryBar } from '../../../components/rules/query_bar';
 import { FieldValueTimeline } from '../../../components/rules/pick_timeline';
+import {
+  Author,
+  BuildingBlockType,
+  License,
+  RiskScoreMapping,
+  RuleNameOverride,
+  SeverityMapping,
+  TimestampOverride,
+} from '../../../../../common/detection_engine/schemas/common/schemas';
```

Comment:
NICE! The common schemas are beginning to make their appearance on the front end.

Reply:
Slowly but surely! I didn't want to make the full-swap as part of this PR in case there was any collateral damage, but may inch a bit closer in `Part II` now that I understand how all our types/schemas are working a bit better 🙂